include/h2o/socket/evloop.h

```diff
@@ -54,6 +54,7 @@ struct st_h2o_timeout_backend_properties_t {
 
 h2o_socket_t *h2o_evloop_socket_create(h2o_evloop_t *loop, int fd, int flags);
 h2o_socket_t *h2o_evloop_socket_accept(h2o_socket_t *listener);
+struct st_h2o_evloop_socket_t *h2o_evloop_socket_accept_ex(h2o_socket_t *listener, size_t sz);
```

Comment:
NB: I understand that we must know the size of the libh2o's socket structure (the real one, not the base `h2o_socket_t`), but this abstraction leak looks a bit inaccurate. I have nothing to propose instead ATM. Just note to think later.